app/models/commodity.rb

```diff
@@ -0,0 +1,29 @@
+class Commodity
+  attr_reader :code
+
+  def initialize(code:, service: :uk)
+    @code = code
+    @service = service
+  end
+
+  def description
+    fetch.data.attributes.description
+  end
+
+  def fetch
+    @fetch ||= begin
+                 commodity = Uktt::Commodity.new(options)
+                 commodity.retrieve
+                 commodity.response
+               end
+  end
+
+  private
+
+  attr_reader :response, :service
+
+  def options
+    @options = OptionBuilder.new(service).call
+      .merge(commodity_id: code)
```

Comment:
For consistency, I think this should be `commodity_code` here.

Reply:
But the option buillder might need to merge in other ids so we'd be losing flexibility for a small naming difference

Reply:
So the Uktt gem expects commodity_id not commodity_code if that wasn't clear

Reply:
Sure you can use
```ruby
{
      host: api_options[service],
      version: 'v2',
      debug: debug?,
      return_json: false,
      commodity_id: commodity_code
    }
```
Then? Also, don't worry about other ids we'll tackle them as they come along. Not sure if in reality, we might need anyhting else but this tbh.

Reply:
Yeah, probably you're right about chapters, sections, etc, not being needed